### DISCUSSION TITLE
Can't get pdfcpu installed / working on my Ubuntu 23.4 laptop.

### DISCUSSION THREAD

**Discussion by Leen58** [pdfcpu/pdfcpu#693] [General]
I installed pdfcpu on Ubuntu 23.4 but i can't get it to run anything. 

I  followed the following procedure without errors:
Using Go Modules

$ git clone https://github.com/pdfcpu/pdfcpu
$ cd pdfcpu/cmd/pdfcpu
$ go install
$ pdfcpu version

i installed go also.
I added go path tomy profile. 

when in the terminal type
pdfcpu version

restarted the laptop.

the response is:
$pdfcpu: command not found.

What am i missing?

**Comment by hhrutter** [comment]
if you are just interested in playing around on the CLI: 

`brew install pdfcpu`

else

`go install github.com/pdfcpu/pdfcpu/cmd/pdfcpu@latest` assuming you have installed go >= 1.20.
and you should not need to setup anything special.

**Comment by Leen58** [comment]
Horst,

Thank you for your quick reply.

After installing brew, i managed to install both the CLI version and it
looks to be working now.

Leen

Op di 29 aug 2023 om 11:54 schreef Horst Rutter ***@***.***>:

> if you are just interested in playing around on the CLI:
>
> brew install pdfcpu
>
> else
>
> go install ***@***.*** assuming you have
> installed go >= 1.20.
> and you should not need to setup anything special.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pdfcpu/pdfcpu/discussions/693#discussioncomment-6851351>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQYSHHWML4AFHJCCUH6KNRTXXW35FANCNFSM6AAAAAA4CVISCE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
M.vr.gr
Leen
